src/main/resources/application.properties

```diff
@@ -1 +1,22 @@
 
+# datasource PostgreSQl
+spring.datasource.platform=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/TechItEasy
+spring.datasource.username=postgres
+spring.datasource.password=Jimlight1990
```

Comment:
Je deelt hiermee je wachtwoord, dus als je die hergebruikt is het misschien handig om voor postgres een andere aan te maken.